libraries/Microsoft.Bot.Builder.AI.QnA/Dialogs/QnAMakerDialog.cs

```diff
@@ -0,0 +1,160 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
```

Comment:
There is no file called "QnAMakerDialogTests.cs". This file needs a corresponding test file. 

I do see a "Microsoft.Bot.Builder.AI.QnA.Tests/QnAMakerTests.cs" file, but that is full of Adaptive tests even it's testing this class.